docs/DeveloperGuide.md

```diff
@@ -464,11 +467,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Use case: UC01- Pair Volunteer and Elderly**
 
+Precondtions: Elderly and Volunteer is already in FL.
+
 **MSS**
 
 1.  User enters the details of elderly and volunteer to be paired into the application.
-2.  FL adds the pair into the database, and feedbacks the successful addition of the pair.
-3.  User see the pair details appear in the pair list.
+2.  FL feedbacks the successful addition of the pair, and shows the new pair.
 
```

Comment:
I feel here should show the FL feedback and user-side feedback, so better maintain the original format

Reply:
Resolved in [a108200](https://github.com/AY2223S2-CS2103T-W12-1/tp/pull/294/commits/a1082000ca6232ebe7eed75ff3d81f9e2a2dac2a)